recipes/flask-turbolinks/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "flask-turbolinks" %}
+{% set version = "0.2.0" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "9086fa5d425db3530efe7eb96d90067f0531158864f0e7130846e534069d6381" %}
+
+package:
+  version: '{{ version }}'
+  name: '{{ name|lower }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.python.org/packages/4d/f3/e7b4ff1fa5c121e81480bf54dcfb8ce61ccab1153cc6d3a00c36f1e7d2a6/Flask-Turbolinks-0.2.0.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
```

Comment:
It seems like this recipe might be a candidate for `noarch: python`. See:

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages

Reply:
True, I will look into that!